docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -156,6 +160,7 @@ Non-Goals listed in this document are intended to scope bound the current v1alph
 3. In service of user story 5, the kubeadm control plane provider must also manage etcd membership via kubeadm as part of scaling down (`kubeadm` takes care of adding the new etcd member when joining).
 4. The control plane provider should provide indicators of health to meet user story 6 and 10. This should include at least the state of etcd and information about which replicas are currently healthy or not. For the default implementation, health attributes based on artifacts kubeadm installs on the cluster may also be of interest to cluster operators.
 5. The control plane provider must be able to upgrade a control plane’s version of Kubernetes as well as updating the underlying machine image on where applicable (e.g. virtual machine based infrastructure).
+6. To address user story 11, the control plane provider must provide rollout strategy to allow control plane to scale in during upgrade using delete-and-create. This strategy should be optional while default strategy is scale out using create-swap-and-delete.
```

Comment:
Assuming a reader without context, I imagine that they could be puzzled but the existence of two upgrade strategies if we don't make this need surface in goals/user stories.
It will be great to have also scale out rollout strategy linked to a user story

Reply:
I made a little change here.